### PR TITLE
Adding edit meeting page

### PR DESCRIPTION
**Pull request by Rodney-QAGeek**
**Issue:** Facilitators are not able to modify their meeting once it's created.

**Solution:** Implement routes and views to allow facilitators to modify their meeting.

### CODE REVIEW

File: routes/squads.js [hunk 1]
```diff
@@ -80,6 +80,67 @@ router.post('/squad/meeting/create/', function (req, res, next) {
...
+router.get('/squad/meeting/:id', function (req, res, next) {
+  var meeting, members;
+  models.SquadMeeting.findById(req.params.id).exec()
+    .then(function(meet){
+      meeting = meet;
+      return models.Member.find({ squad: meeting.squad }).exec();
+    })
+    .then(function(mem) {
+      members = mem;
+      return models.Squad.findById(meeting.squad).exec();
```
Comment (makinde) [inline routes/squads.js]: You can get rid of this query by using a `.populate()` on the first query to fill in the squad of the meeting.

**Comment by makinde** [comment]
Looks really great. Had one small simplification inline. After that I can land this baby! Major Diff here. Major Key :)

**Comment by Rodney-QAGeek** [comment]
@makinde All fixed up! Thanks for the suggestion!

**Comment by makinde** [comment]
BOOM! Nice work here. This'll make things much easier on facilitators!

**Pull request merged**